docs/source/fates_tech_note.rst

```diff
@@ -2064,40 +2065,205 @@ where :math:`r_{s}` is leaf stomatal resistance (s m\ :math:`^2`
+|                   |                          | m\ :math:`^{2}`  |            |
+|                   |                          | s\ :math:`^{-1}` |            |
++-------------------+--------------------------+------------------+------------+
+| :math:`b_{ft}`    | Minimum stomatal         | :math:`\mu`\mol  |*ft*        |
+|                   | conductance or the       | m\ :math:`^{2}`  |            |
+|                   | cuticular conductance    | s\ :math:`^{-1}` |            |
++-------------------+--------------------------+------------------+------------+
+| :math:`\beta_{sw}`| Soil water stress factor | none             |            |
++-------------------+--------------------------+------------------+------------+
+| :math:`D_{s}`     | Vapor pressure deficit at| kPa              |            |
+|                   | the leaf surface         |                  |            |
++-------------------+--------------------------+------------------+------------+
+| :math:`m_{ft}`    | Stomatal slope           | kPa\             |            |
+|                   |                          | :math:`^{0.5}`   |*ft*        |
++-------------------+--------------------------+------------------+------------+
+| :math:`A_{n}`     | Leaf net photosynthesis  | :math:`\mu`\mol  |            |
+|                   |                          | :math:`CO_{2}` m\|            |
+|                   |                          | :math:`^{-2}` s\ |            |
+|                   |                          | :math:`^{-1}`    |            |
++-------------------+--------------------------+------------------+------------+
+| :math:`C_{s}`     | :math:`CO_{2}` partial   | Pa               |            |
+|                   | pressure at the leaf     |                  |            |
+|                   | surface                  |                  |            |
++-------------------+--------------------------+------------------+------------+
+| :math:`P_{atm}`   | Atmospheric pressure     | Pa               |            |
++-------------------+--------------------------+------------------+------------+
+
+In both models leaf resistance is converted from units of s m\ :math:`^2 \mu`
 mol\ :math:`^{-1}` to s m\ :math:`^{-1}` as: 1 s m\ :math:`^{-1}` =
 :math:`1\times 10^{-9}`\ R\ :math:`_{\rm{gas}} \theta_{\rm{atm}}P_{\rm{atm}}`
 (:math:`\mu`\ mol\ :math:`^{-1}` m\ :math:`^{2}` s), where
 R\ :math:`_{gas}` is the universal gas constant (J K\ :math:`^{-1}`
 kmol\ :math:`^{-1}`) and :math:`\theta_{atm}` is the atmospheric
 potential temperature (K).
 
+Both :math:`b_{ft}` and :math:`m_{ft}` are PFT-specific parameters. The default
+values for the Ball-Berry and Medlyn stomatal conductance model representations
+are provide below:
+
+.. raw:: latex
+
+   \captionof{table}{Variables use in the Medlyn equation}
+
++-----------------------------+--------------------------+----------------------+
```

Comment:
Is it a bad idea to embed parameter defaults in the docs? Will require keeping in line with any changes in the params.  But helpful as it exposes these to the user. That said the text version of the param file is easy to view on Git.  So I am between two minds here